rero_ils/modules/documents/serializers/marc.py

```
@@ -183,22 +282,99 @@ def transform_records(
+        MARC21_REC (str): MARC 21 XML Schema namespace URI.
+
+    See Also:
+        - SRU Protocol: https://www.loc.gov/standards/sru/
+        - MARC 21 XML: https://www.loc.gov/standards/marcxml/
     """
 
     MARC21_ZS = "http://www.loc.gov/zing/srw/"
     MARC21_REC = "http://www.loc.gov/MARC21/slim"
     """MARCXML XML Schema"""
 
     def dumps_etree(self, total, records, sru, xslt_filename=None, prefix=None):
```

Ruff (0.15.1)

[warning] Unused method argument: `xslt_filename`

(ARG002)